scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -1185,6 +1201,31 @@ class MicrobeEditor{
                 GetThriveGame().playerMovedToPatch(targetPatch);
             }
 
+            // Change species name
+            if(newName.findFirst(" ") > -1){
+                playerSpecies.genus = newName.split(" ")[0];
+                playerSpecies.epithet = newName.split(" ")[1];
+                LOG_INFO("MicrobeEditor: Player species name is now " + playerSpecies.genus + " " + playerSpecies.epithet);
+            } else {
+                LOG_INFO("MicrobeEditor: Invalid newName: " + newName);
```

Comment:
Is this requirement shown to the user? Could it be made so that the name input box would turn red if the name is not acceptable?